### ISSUE TITLE
Icon request: PHP template engines (twig, blade) & Magento 2 templates (phtml)

### ISSUE THREAD

**Issue by emanueljacob** [chadalen/vscode-jetbrains-icon-theme#26]
Hey there,

first things first: Thanks for this clean icon theme, it's almost exactly what i needed.

I wonder if you could add PHP specific icons, especially blade, twig (template engines) and phtml (Magento 2 templates):

Icons for blade and twig come from [https://jetbrains.design/intellij/resources/icons_list/ ](https://jetbrains.design/intellij/resources/icons_list/ ), search for:
- "blade"
- "twig"

The pthml is just php code mixed with html, so it can and should be (imo) referenced to "php".
- "phtml"

I already prepared the json modification for you, so you just have to check, copy & paste it, download and push the icons from the url above together with the json changes and that should be it.

```json
...
    "iconDefinitions": {
        ...
        "file_type_blade": {
            "iconPath": "./icons/blade.svg"
        },
        "file_type_twig": {
            "iconPath": "./icons/twig.svg"
        },
        ...
    }
    ...
    "fileNames": {
        ...
        "blade.php": "file_type_blade",
        "twig": "file_type_twig",
        "phtml": "file_type_php",
        ...
    }
...
```

Thanks!

**Comment by chadalen** [comment]
Hey thanks for creating an issue. This looks good I'll take a look at it later today. 🙂

**Comment by chadalen** [comment]
@emanueljacob Questions

1. Is phtml the extension of the file? If so that would go under
"fileExtensions" instead of "fileNames"

2. Is "twig" a file or folder? I'm assuming it's a folder? If so that would go under "folderNames" instead of "fileNames".

**Comment by chadalen** [comment]
@emanueljacob See PR https://github.com/chadalen/vscode-jetbrains-icon-theme/pull/27 

It's going to look like this
![https://i.imgur.com/OWwBFUd.png](https://i.imgur.com/OWwBFUd.png)
![https://i.imgur.com/XqRFnZw.png](https://i.imgur.com/XqRFnZw.png)

**Comment by emanueljacob** [comment]
Thanks, thats exactly how it should look like. ☺️

**Comment by chadalen** [comment]
Closing this issue. This has been added in release 1.0.8

**Comment by emanueljacob** [comment]
@chadalen 
Im sorry, its not exactly correct. I made a mistake in my original data example and only checked your screenshot.

Of course blade templates are not specific files with an exact phrase `blade.php` but rather files with an extension of `blade.php` i.e. a file like this: `foo-bar-baz.blade.php`.

I've already prepared a PR for you: https://github.com/chadalen/vscode-jetbrains-icon-theme/pull/28. Sorry for the inconvenience.
